rs/crypto/internal/crypto_lib/threshold_sig/canister_threshold_sig/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "ic-crypto-internal-threshold-sig-ecdsa"
```

Comment:
My personal preference would be to keep consistency with the folder names. It would be possible in general to have better names while having consistency, but in an offline discussion we decided to consider this as a next step and this PR is about renaming the `tecdsa` crate s.t. the name makes sense in general and eventually improve later.